internal/service/instance/nginx_instance_service_test.go

```diff
@@ -70,6 +75,20 @@ const (
 
 func TestGetInstances(t *testing.T) {
 	ctx := context.Background()
+	tempDir := t.TempDir()
+	modulePath := tempDir + "/usr/lib/nginx/modules"
+
+	helpers.CreateDirWithErrorCheck(t, modulePath)
+	defer helpers.RemoveFileWithErrorCheck(t, modulePath)
+
+	testModule := helpers.CreateFileWithErrorCheck(t, modulePath, "test.so")
+	defer helpers.RemoveFileWithErrorCheck(t, testModule.Name())
+
+	plusArgs := fmt.Sprintf(plusConfigArgs, modulePath)
+	ossArgs := fmt.Sprintf(ossConfigArgs, modulePath)
+	noModuleArgs := fmt.Sprintf(ossConfigArgs, t.TempDir()+"\"/usr/lib/nginx/modules\"")
```

Comment:
nit/optional:

This is a good place to use `path.Join` so escaping slashes is no longer required. (Extra import required too)
```suggestion
	noModuleArgs := path.Join(ossConfigArgs, t.TempDir(), "/usr/lib/nginx/modules")
```